### PR TITLE
ENH: Conversion from *AcqKnowledge* to BIDS

### PR DESCRIPTION
**Pull request by oesteban**
This PR concludes the conversion from *AcqKnowledge* into BIDS:

- [x] Sessions 1-24 have been added to the datalad dataset on curnagl with physio. Exceptions
    - Sessions 2, 12 and 14, which we skipped.
    - Sessions 11 and 13, which have multiple files per session and I will address those later (#324)
    - Session 4, which has *AcqKnowledge* file but it doesn't seem in a good state (#325).
- [x] Update SOPs accordingly, including a Jupyter Notebook showing the conversion.

Relates-to: #120.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -66,12 +67,17 @@ plugins:
...
+      ignore:
+        - "*.py"
+        - "assets/code/physioconv/.ipynb_checkpoints/Physio to BIDS conversion-checkpoint.ipynb"
+        - "assets/code/physioconv/Physio to BIDS conversion.ipynb"
+  # - mkpdfs:
+  #     company: Lausanne University Hospital (CHUV)
+  #     author: The Axon Lab
+  #     toc_title: Table of Contents
+  #     design: design/report.css
+  #     output_path: pdf/documentation.pdf
```
Comment (oesteban) [inline mkdocs.yml:80]: Let me sneak this in, otherwise development is super slow

**Comment by oesteban** [comment]
Jupyter notebook rendered: https://output.circle-artifacts.com/output/job/286f0227-748d-47af-ba4f-91a4b8ecb3ec/artifacts/0/_SOPs_preview/data-management/physio-to-bids/index.html

File: mkdocs.yml [hunk 2]
```diff
@@ -66,12 +67,17 @@ plugins:
...
-      company: Lausanne University Hospital (CHUV)
-      author: The Axon Lab
-      toc_title: Table of Contents
-      design: design/report.css
-      output_path: pdf/documentation.pdf
+  - mkdocs-jupyter:
+      ignore:
+        - "*.py"
+        - "assets/code/physioconv/.ipynb_checkpoints/Physio to BIDS conversion-checkpoint.ipynb"
+        - "assets/code/physioconv/Physio to BIDS conversion.ipynb"
```
Comment (acionca) [inline mkdocs.yml]: Should we replace the spaces with underscore in the filenames ?
```suggestion
        - "assets/code/physioconv/.ipynb_checkpoints/Physio_to_BIDS_conversion-checkpoint.ipynb"
        - "assets/code/physioconv/Physio_to_BIDS_conversion.ipynb"
```

File: docs/data-management/physio-to-bids.ipynb [hunk 3]
```diff
@@ -0,0 +1 @@
+../../code/physioconv/Physio to BIDS conversion.ipynb
```
Comment (acionca) [inline docs/data-management/physio-to-bids.ipynb]: Should we replace the space with underscores ?
```suggestion
../../code/physioconv/Physio_to_BIDS_conversion.ipynb
```

**Review by acionca — COMMENTED** [review]
I just did a quick check for now, here are some suggestions:

- I would suggest renaming the notebook with underscores instead of spaces.

- Also, I tried to `mkdocs serve` and got the error below, could it be a missing dependancy ?

```shell
jinja2.exceptions.UndefinedError: 'mermaid_js' is undefined
```

EDIT: Looks like there's a known bug with `mkdocs-jupyter` 0.24.3+ (see https://github.com/danielfrg/mkdocs-jupyter/issues/176) – I would suggest to force using 0.24.2 which worked for me.

- On my local `mkdocs serve`, the latex math rendering with `$` did not render properly (e.g., `$t=0$`)

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 codespell
 markdown-include
 mkdocs
+mkdocs-jupyter
```
Comment (acionca) [inline requirements.txt]:
```suggestion
mkdocs-jupyter==0.24.2
```

**Comment by oesteban** [comment]
> * Also, I tried to `mkdocs serve` and got the error below, could it be a missing dependancy ?
> 
> 
> ```shell
> jinja2.exceptions.UndefinedError: 'mermaid_js' is undefined
> ```

I think you need material >= 8.2.0 (https://squidfunk.github.io/mkdocs-material/changelog/#8.2.0)



> * On my local `mkdocs serve`, the latex math rendering with `$` did not render properly (e.g., `$t=0$`)

Let's open an issue for this somewhere else

**Pull request merged**